libjlm/include/jlm/opt/unroll.hpp

```diff
@@ -225,6 +233,13 @@ class unrollinfo final {
 	jive::argument * idv_;
 };
 
+
+/**
+* Try to unroll the given theta.
+*
+* \param theta The theta to attempt the unrolling on.
+* \param factor The number of times to unroll the loop.
```

Comment:
Can we add an example here to specify what factor means? Like if the factor is 2, then the loop body is duplicated in the unrolled loop (if that is actually true).